plugins/connection/lxd.py

```diff
@@ -71,26 +71,30 @@ def __init__(self, play_context, new_stdin, *args, **kwargs):
         if self._play_context.remote_user is not None and self._play_context.remote_user != 'root':
             self._display.warning('lxd does not support remote_user, using container default: root')
 
+    def _host(self):
+        """ translate remote_addr to lxd (short) hostname """
+        return self.get_option("remote_addr").split(".")[0]
```

Comment:
Not that performance here matters that much, but splitting more than once is not necessary:
```suggestion
        return self.get_option("remote_addr").split(".", 1)[0]
```